FinSearchUnified/Subscriber/Widgets.php

```diff
@@ -33,4 +116,25 @@ public function beforeListingCountAction(Enlight_Hook_HookArgs $args)
             }
         }
     }
+
+    /**
+     * @param Enlight_Controller_Request_RequestHttp $request
+     *
+     * @return string
+     * @throws Exception
+     */
+    private function parseReferUrl($request)
```

Comment:
Rename this function to `parseReferer` as you already know by the name "referer" that it is an URL.